schema.sql

```diff
@@ -67,6 +67,9 @@ CREATE TABLE visits (
 /*ALTER TABLE visits 
 ADD visit_date DATE;*/
 
+ALTER TABLE owners 
+ADD email varchar(120);
+
 
 
 
```

Comment:
- [ ] We were supposed to see the solution of this challenge in the schema.sql file 
  - [ ] The expected solution is to add indexes. Adding indexes for FOREIGN KEYS or on columns frequently used in queries is one of the basic solutions for the performance problems so would you try it out.

Reply:
thanks for the review.
we have made the required changes